subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_verification.adoc

```diff
@@ -0,0 +1,849 @@
+- `name`, the name of the artifact to trust
+- `version`, the version of the artifact to trust
+- `file`, the name of the artifact _file_ to trust
+- `regex`, a boolean saying if the `group`, `name`, `version` and `file` attributes need to be interpreted as regular expressions (defaults to `false`)
+
+In the example above it means that the trusted artifacts would be artifacts in `com.mycompany` but not `com.mycompany.other`.
+To trust all artifacts in `com.mycompany` and all subgroups, you can use:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata>
+   <configuration>
+      <trusted-artifacts>
+         <trust group="^com[.]mycompany($|([.].*))" regex="true"/>
+      </trusted-artifacts>
+   </configuration>
+</verification-metadata>
+----
+
+[[sec:trusting-several-checksums]]
+=== Trusting multiple checksums for an artifact
+
+It's quite common to have _different checksums for the same artifact_ in the wild.
+How is that possible?
+Despite progress, it's often the case that developers publish for example to Maven Central and JCenter separately, using different builds.
+In general, this is not a problem but sometimes it means that the metadata files would be different (different timestamps, additional whitespaces, ...).
+Add to this that your build may use several repositories or repository mirrors and it makes it quite likely that a single build can "see" different metadata files for the same component!
+In general, it's not malicious (but you **must** verify that the artifact is actually correct), so Gradle lets you declare the additional artifact checksums.
+For example:
+
+[source,xml]
+----
+      <component group="org.apache" name="apache" version="13">
+         <artifact name="apache-13.pom">
+            <sha256 value="2fafa38abefe1b40283016f506ba9e844bfcf18713497284264166a5dbf4b95e">
+               <also-trust value="ff513db0361fd41237bef4784968bc15aae478d4ec0a9496f811072ccaf3841d"/>
+            </sha256>
+         </artifact>
+      </component>
+----
+
+You can have as many `also-trust` entries as needed, but in general you shouldn't have more than 2.
+
+[[sec:skipping-javadocs]]
+=== Skipping Javadocs and sources
+
+By default Gradle will verify _all_ downloaded artifacts, which includes Javadocs and sources.
+In general this is not a problem but you might face an issue with IDEs which automatically try to download them during import: if you didn't set the checksums for those too, importing would fail.
+
+To avoid this, you can configure Gradle to trust automatically all javadocs/sources:
+
+[source,xml]
+----
+<trusted-artifacts>
+   <trust file=".*-javadoc[.]jar" regex="true"/>
+   <trust file=".*-sources[.]jar" regex="true"/>
+</trusted-artifacts>
+----
+
+[[sec:verification-metadata-hygiene]]
+=== Cleaning up the verification file
+
+The dependency verification metadata file can grow up easily.
+Gradle will not automatically remove _unused_ dependencies from this file.
```

Comment:
`dependencies` -> `entries` ?

`trusted-artifacts ` does not look like `dependencies`, however, its entries may become unused as well.